scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -137,8 +140,7 @@ task CollectAllelicCounts {
     Int? preemptible_attempts
     Int? disk_space_gb
 
-    # Mem is in units of GB but our command and memory runtime values are in MB
```

Comment:
Somewhere it should made clear that the user provided `mem` input should be defined in units of GB

Reply:
Any objection to simply using `*_mem_gb` and `*_mem_mb` everywhere?  I'd rather not duplicate this comment everywhere. 
 @LeeTL1220 @davidbenjamin?

Reply:
@samuelklee Fine by me...

Reply:
Done.

Reply:
> Any objection to simply using *_mem_gb and *_mem_mb everywhere?

That is sufficiently self-documenting for my tastes.